amethyst_network/src/network_socket.rs

```diff
@@ -85,15 +82,15 @@ where
     }
 
     /// Start a thread to send all queued packets.
-    fn start_sending(sender: Arc<SendHandler>) -> Sender<InternalSocketEvent<E>> {
+    fn start_sending(sender: crossbeam_channel::Sender<Packet>) -> Sender<InternalSocketEvent<E>> {
```

Comment:
since we are importing this above, can we import and use `Sender<Packet>` and same thing below on 107.